### PR TITLE
Implement plugin inspect command

### LINKED ISSUE #2323: Plugin info/inspect command
`singularity plugin <info/inspect> <name>`

This command should be used to provide any information about the named plugin. Could either be `inspect` or `info`, I'm rather indifferent. `inspect` might make more since, as `singularity inspect` is already a command for retrieving information about an image. This information can be retrieved from a plugin "manifest"

### PR DESCRIPTION
**Pull request by mem**
- Fixes #2323

Signed-off-by: Marcelo E. Magallon <marcelo@sylabs.io>

### CODE REVIEW

File: cmd/internal/cli/plugin_inspect_linux.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package cli
```
Comment (WestleyK) [inline cmd/internal/cli/plugin_inspect_linux.go:6]: Need copyright here. 

```go
// Copyright (c) 2018-2019, Sylabs Inc. All rights reserved.
// This software is licensed under a 3-clause BSD license. Please consult the
// LICENSE.md file distributed with the sources of this project regarding your
// rights to use or distribute this software.
```

File: internal/app/singularity/plugin_inspect_linux.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package singularity
```
Comment (WestleyK) [inline internal/app/singularity/plugin_inspect_linux.go:6]: Here too. 

```go
// Copyright (c) 2018-2019, Sylabs Inc. All rights reserved.
// This software is licensed under a 3-clause BSD license. Please consult the
// LICENSE.md file distributed with the sources of this project regarding your
// rights to use or distribute this software.
```

File: cmd/internal/cli/plugin_inspect_linux.go (hunk 1 above)
Comment (mem) [reply cmd/internal/cli/plugin_inspect_linux.go:6]: ah, thanks!

File: cmd/internal/cli/plugin_inspect_linux.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+	"github.com/spf13/cobra"
+	"github.com/sylabs/singularity/docs"
+	"github.com/sylabs/singularity/internal/app/singularity"
+	"github.com/sylabs/singularity/internal/pkg/buildcfg"
+	"github.com/sylabs/singularity/internal/pkg/sylog"
+)
+
+// PluginInspectCmd displays information about a plugin
+var PluginInspectCmd = &cobra.Command{
```
Comment (WestleyK) [inline cmd/internal/cli/plugin_inspect_linux.go:19]: Do you need to add:
```go
Args: cobra.ExactArgs(1),
DisableFlagsInUseLine: true,
```
*Or whatever cobra likes...*

Becuse you use `args[0]` in `singularity.InspectPlugin()`.

Comment (bauerm97) [reply cmd/internal/cli/plugin_inspect_linux.go:19]: This already exists a few lines down ;) `line 34`

Comment (WestleyK) [reply cmd/internal/cli/plugin_inspect_linux.go:19]: :+1:

**Review by bauerm97 — APPROVED** [review]
LGTM

**Review by WestleyK — APPROVED** [review]
Looks Good to Me :+1:

**Pull request merged**